### ISSUE TITLE
Trying to reproduce results

### ISSUE THREAD

**Issue by pranav-asthana** [XiangFeng66/SRGS#12]
Hi,

Congratulations on your project and results. I am trying to use your work for a project of mine. I noticed some lines in the train script are commented so the loss is effectively only the sr loss
```
loss_lr = (1.0 - opt.lambda_dssim) * Ll1_lr + opt.lambda_dssim * (1.0 - ssim(image, gt_image))
# loss = (1-alpha) * loss_sr + alpha * loss_lr
# loss = 0.4 * loss_sr + 0.6 * loss_lr
loss = loss_sr
```

Is this what is used to produce the results in the paper? I also tried commenting  the last line above and setting the loss as `loss = 0.4 * loss_sr + 0.6 * loss_lr` but I was not able to get the same result. Is there a different weight used for computing the loss? Is the result in the paper from taking an aggregate over several runs of the training?

Looking forward to your clarification.

Thanks!

**Comment by XiangFeng66** [comment]
@pranav-asthana  Hello, thank you for your attention to SRGS. The combination of lr and hr losses is the key to obtaining the paper data. I would like to ask which dataset you are referring to. Our method has been integrated by many subsequent works. It is simple and easy to use. You can refer to sequence matters, etc.
